packages/office-addin-dev-certs/src/uninstall.ts

```diff
@@ -13,6 +13,8 @@ function getUninstallCommand(machine: boolean = false): string {
          return `powershell -ExecutionPolicy Bypass scripts\\uninstall.ps1 ${machine ? "LocalMachine" : "CurrentUser"} '${defaults.certificateName}'`;
       case "darwin": // macOS
          return `sudo security delete-certificate -c '${defaults.certificateName}'`;
+      case "linux":
+         return `sudo rm /usr/local/share/ca-certificates/${defaults.certificateName} && sudo /usr/sbin/update-ca-certificates`
```

Comment:
When you move ca.crt file to /usr/local/share/ca-certificates shouldn't you be deleting ca.crt?. So this should be sudo rm /usr/local/share/ca-certificates/${defaults.caCertificateFileName}

Reply:
I'm not sure I understand the question. 

Also I think the `update-ca-certificates` statement here should be called with `--fresh`.

Reply:
defaults.certificateName value is "Developer CA for Microsoft Office Add-ins" not ca.crt. So rm won't work

Reply:
Ah, thanks. Of course. That's what I get for doing anything after 9pm. Will change.